.travis.yml

```diff
@@ -1,21 +1,18 @@
 sudo: required
```

Comment:
I swear the docs say you need:
```
services:
  - docker
```
But this appears to work??  I'd still add it though, better not to rely on undocumented behaviour I guess.